op-program/Makefile

```diff
@@ -47,7 +47,10 @@ op-program-client-mips64-interop:
 op-program-client-riscv:
 	env GO111MODULE=on GOOS=linux GOARCH=riscv64 go build -v -gcflags="all=-d=softfloat" -ldflags "$(PC_LDFLAGSSTRING)" -o ./bin/op-program-client-riscv.elf ./client/cmd/main.go
 
-reproducible-prestate:
+check-custom-chains:
+	env GO111MODULE=on go run ./host/cmd configs check-custom-chains
+
+reproducible-prestate: check-custom-chains
```

Comment:
Very clever.

Reply:
This will cause a build of op-program-host and then I _think_ because the actual `op-program-host` has different `-ldflags` and `CGO_ENABLED=0` etc I think it will wind up having to rebuild it again.  Would it be better to have this depend on `op-program-host` target and then run `./bin/op-program configs check-custom-chains`?

Go tooling may be smarter than I'm expecting here though.  The kurtosis startup time is very sensitive to `make reproducible-prestate` duration so keen to keep this from slowing it down further.